src/Opserver.Web/ContentSource/js/Scripts.js

```diff
@@ -1053,9 +1053,29 @@ Status.Exceptions = (function () {
             store: options.store,
             group: options.group,
             log: options.log,
-            sort: options.sort
+            sort: options.sort,
+            url: options.url,
+            host: options.host,
+            startDate: options.startDate,
+            endDate: options.endDate
         };
 
+        var filterOptions = {
+            modalClass: 'modal-md',
+            buttons: {
+                "Apply Filters": function (e) {
+                    $(this).find('#exceptionFiltersForm').submit();
```

Comment:
We should be able to stay from IDs here with `.find('form')` and such - can we eliminate IDs please? (These create problems vs. JS when changed and such).

Reply:
Done